src-vue/src/utils/thunderstore/version.ts

```diff
@@ -0,0 +1,32 @@
+import {ThunderstoreMod} from "../../../../src-tauri/bindings/ThunderstoreMod";
+import {NorthstarMod} from "../../../../src-tauri/bindings/NorthstarMod";
+import {store} from "../../plugins/store";
+
+/**
+ * Strips off a Thunderstore dependency string from its version
+ * (e.g. "taskinoz-WallrunningTitans-1.0.0" to
+ * "taskinoz-WallrunningTitans").
+ **/
+function getThunderstoreDependencyStringPrefix(dependency: string): string {
```

Comment:
I'll probably move this to backend functions at some point but for the sake of getting this out ASAP I'm fine with having it in frontend for now ^^